File: Shift Left/Shift Left.js
function shiftLeft(s, t){
  let common = 0;
  while(common < s.length && common < t.length && (s[s.length-1-common] === t[t.length-1-common])){
    common++;
  }
  return s.length-common + t.length-common
}

File: Shift Left/Shift Left.dart
import 'dart:math';
int shiftLeft(String a, String b){
  int minVal = [a.split("").length,b.split("").length].reduce(min);
  List<String> a_r = a.split('').reversed.toList();
  List<String> b_r = b.split('').reversed.toList();
  int common = 0;
  for(int i=0;i<minVal;i++){
    if(a_r[i]==b_r[i]){
      common+=1;
    }else{
      break;
    }
  }
  return (a.length-common)+(b.length-common);
}
